docroot/sites/all/themes/custom/wsdot_classic/css/styles.css

```diff
@@ -73,7 +73,7 @@ a.facebook{background: url(../images/back/facebook.png) top left no-repeat;paddi
 .topnav a:hover{text-decoration:underline;color:#ffffff;}
 .topnav a:visited{color:#ffffff;}
 .topnav ul {list-style:none;}
-.topnav li{list-style:none;font-size:.75em; float:right; border-left:1px dotted #ffffff;padding:0em .6em 0em .6em;}
+.topnav li{list-style:none;font-size:11px;float:right;border-left:1px dotted #ffffff;padding:0em .6em 0em .6em;}
```

Comment:
It renders the same on both sites, whereas em sizes for .topnav li and .mainnav ul li a do not.